dashboard/test/factories/factories.rb

```diff
@@ -1550,6 +1562,14 @@
     end
   end
 
+  factory :code_review do
```

Comment:
`association` would work here I think but then it would create all the related objects, which may not be ideal. I think we tend to lean towards creating the objects for the associations but our dashboard tests are quite slow and this doesn't help :)

Reply:
Side note: Maureen and I have been talking a bit about how much a code_review really needs to know about the script/level/project that it's attached to.  I think it's possible that code_review just knows that it's attached to _something_ (that we identify via project id + version) but otherwise don't need to verify its contents or perhaps even that it exists.

I know this all maybe sounds a little strange because it's very un-rails-y and seems bad from a data integrity standpoint, but this is common when separating microservices.  I know we're not fully moving in that direction yet (and I've used associations to `user` where needed, for example), but I'm also intentionally not adding associations that aren't needed.